index.html

```diff
@@ -34,22 +38,212 @@ <h1>Hey There.
                     I’m Aneal</h1>
             </div>
             <div class="supporting-text">
-                <p>I’am a Software Developer</p>
+                <h5>I’am a Software Developer</h5>
             </div>
             <div class="description">
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hestiate to contact me.</p>
+                <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</h5>
             </div>
             <div class="social-media">
                 <ul class="socials">
-                    <li><i class="fa-brands fa-github"></i></li>
-                    <li><i class="fa-brands fa-linkedin"></i></li>
-                    <li><i class="fa-brands fa-angellist"></i></li>
-                    <li><i class="fa-brands fa-twitter"></i></li>
-                    <li><i class="fa-solid fa-m"></i></li>
+                    <li class="ss"><i class="fa-brands fa-github"></i></li>
+                    <li class="ss"><i class="fa-brands fa-linkedin"></i></li>
+                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
+                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
+                    <li class="ss"><i class="fa-brands fa-twitter"></i></li>
+                    <li class="ss"><i class="fa-solid fa-m"></i></li>
                 </ul>
             </div>
         </section>
+
+        <section class="work-section">
+            <div class="title">
+                <h1 class="title-text">My Recent Works</h1>
+            </div>
+
+             <!--frame 44-->
+            <div class="frame">
+                <div class="featured-img">
+                    <div class="img-holder"></div>
+                    <div class="right-block">
```

Comment:
Please ensure that the main container of the project has a display:grid according to the project requirement.